packages/@o3r/schematics/src/tasks/package-manager/add-dev-dependency.ts

```diff
@@ -5,6 +5,7 @@ import { NodePackageInstallTaskOptions } from './interfaces';
 
 /**
  * Install new dev dependency on your package
+ * @deprecated use {@link setupDependencies} instead, will be removed in V11
```

Comment:
v12?

Reply:
nop, will not wait 1 year to remove a schematic helper